packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,217 @@
+import type { ReactChild, ReactElement } from 'react';
+import { useState } from 'react';
+
+import StyledAlert from './StyledAlert';
+import { AlertIcon, CloseIcon } from '~components/Icons';
+import { makeAccessible } from '~utils';
+import { IconButton } from '~components/Button/IconButton';
+import Box from '~components/Box';
+import { Heading, Text } from '~components/Typography';
+import BaseButton from '~components/Button/BaseButton';
+import { BaseLink } from '~components/Link/BaseLink';
+
+type Nullable<Type> = Type | null;
+
+type PrimaryAction = {
+  text: string;
+  onClick: () => void;
+};
+
+type SecondaryActionButton = {
+  text: string;
+  onClick: () => void;
+};
```

Comment:
Both are same, why do we keep them separate?

Reply:
Both signify different things so I feel it's better to keep them separate even if right now they've same types.